packages/bot-web-ui/src/stores/load-modal-store.js

```diff
@@ -29,30 +29,26 @@ export default class LoadModalStore {
     @action.bound
     setActiveTabIndex(index) {
         this.active_index = index;
+        this.abs_index = this.root_store.ui.is_mobile ? this.active_index + 1 : this.active_index;
```

Comment:
Rather than creating this variable, could we assign the correct value to `this.active_index` one line above this one? It'll be a bit clearer as to what we're referring to. e.g.:

```suggestion
this.active_index = this.root_store.ui.is_mobile ? index + 1 : index;
```